Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,39 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int numPeople = 2;
+
+            while(numPeople > 0)
+            {
+                Console.Write("Enter name: ");
+                string name = Console.ReadLine();
+
+                Console.Write("Enter surname: ");
+                string surname = Console.ReadLine();
+
+                Console.Write("Enter age: ");
+                int age = int.Parse(Console.ReadLine());
+
+                Console.Write("Enter weight (in kg): ");
+                double weight = double.Parse(Console.ReadLine());
+
+                Console.Write("Enter height (in cm): ");
+                double height = double.Parse(Console.ReadLine());
+
+                Console.WriteLine($"{name} {surname} is {age} years old, his weight is {weight} kg and his height is {height} cm.");
+
+                BMIPrinter(weight, height);
+
+                numPeople--;
+            }
+
+            Console.ReadKey();
+        }
+
+        static void BMIPrinter(double weight, double height)
```

Comment:
This actually doesn't just print Bmi, it also calculates it.
I would say it's better to split it into two functions (maybe just one)
But don't couple printing with calculating- the two are very different things.

Reply:
Agreed.